docs/copied-from-beats/dashboards.asciidoc

```diff
@@ -0,0 +1,83 @@
+//////////////////////////////////////////////////////////////////////////
+//// This content is shared by all Elastic Beats. Make sure you keep the
+//// descriptions here generic enough to work for all Beats that include
+//// this file. When using cross references, make sure that the cross
+//// references resolve correctly for any files that include this one.
+//// Use the appropriate variables defined in the index.asciidoc file to
+//// resolve Beat names: beatname_uc and beatname_lc.
+//// Use the following include to pull this content into a doc file:
+//// include::../../libbeat/docs/dashboards.asciidoc[]
+//////////////////////////////////////////////////////////////////////////
+
+
+{beatname_uc} comes packaged with example Kibana dashboards, visualizations,
+and searches for visualizing {beatname_uc} data in Kibana. Before you can use
+the dashboards, you need to create the index  pattern, +{beat_default_index_prefix}-*+, and
+load the dashboards into Kibana. To do this, you can either run the `setup`
+command (as described here) or
+<<configuration-dashboards,configure dashboard loading>> in the
++{beatname_lc}.yml+ config file.
+
+NOTE: Starting with {beatname_uc} 6.0.0, the dashboards are loaded via the Kibana API.
+This requires a Kibana endpoint configuration. You should have configured the
+endpoint earlier when you
+<<{beatname_lc}-configuration,configured {beatname_uc}>>. If you didn't,
+configure it now.
```

Comment:
Actually the config part comes *after*. this, so saying `you should have configured the endpoint earlier` is a bit confusing. 
I suggest to completely remove this `NOTE` section. Instead only link to the `configure dashboard loading`, which itself should link to the Kibana API setup section.